content/src/Blip.Api.Template/Models/NLPContentProviderModels/NLPAnalyseResponse.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Threading.Tasks;
+
+namespace Blip.Api.Template.Models.NLPContentProviderModels
+{
+    [DataContract]
+    public class NLPAnalyseResponse
```

Comment:
This should be removed alonside any mention of ContentProviderAPI, and moved to an independent package